assimp: fail to compile in personal project

I have a project to test VitaGL and I use assimp to load the 3D meshes. With the update to assimp v5.3.1 the compilation breaks. It sais the there undefined reference to `fseeko64`, `ftello64` and `fseeko64`. This is because in the new code there is no `#else` clause for the rest of cases in file ioapi.h. 

I have been able to fix it with the next patch:

```
diff --git a/contrib/unzip/ioapi.h b/contrib/unzip/ioapi.h
index ae9ca7e83..50978316d 100644
--- a/contrib/unzip/ioapi.h
+++ b/contrib/unzip/ioapi.h
@@ -50,7 +50,7 @@
 #define ftello64 ftell
 #define fseeko64 fseek
 #else
-#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__)
+#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) || defined(__vita__)
 #define fopen64 fopen
 #define ftello64 ftello
 #define fseeko64 fseeko
```

I have used the above solution because is simpler.

--- a/assimp/VITABUILD
+++ b/assimp/VITABUILD
@@ -4,6 +4,7 @@ pkgrel=1
 url="https://www.assimp.org/"
 source=("https://github.com/assimp/assimp/archive/refs/tags/v${pkgver}.tar.gz" "assimp-no-pic.patch")
 sha256sums=("66dfbaee288f2bc43172440a55d0235dfc7bf885dda6435c038e8000e79582cb" "434e8fb03c3af8635db11ded994ebe907083cdd5e593b860663970f0d58e2414")
+depends=('minizip zlib')
 
 prepare() {
   cd assimp-$pkgver
@@ -15,7 +16,7 @@ build() {
   mkdir build && cd build
 
   cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=$VITASDK/share/vita.toolchain.cmake -DCMAKE_INSTALL_PREFIX=$prefix \
-    -DASSIMP_NO_EXPORT=ON -DBUILD_SHARED_LIBS=OFF -DASSIMP_BUILD_ZLIB=ON -DASSIMP_BUILD_TESTS=OFF -DASSIMP_WARNINGS_AS_ERRORS=OFF
+    -DASSIMP_NO_EXPORT=ON -DBUILD_SHARED_LIBS=OFF -DASSIMP_BUILD_TESTS=OFF -DASSIMP_WARNINGS_AS_ERRORS=OFF
 
   make -j$(nproc)
 }

